WTWSMS/common/religions/00_religions.txt

```diff
@@ -2516,12 +2558,13 @@ pagan_group = {
 		female_temple_holders = yes
 		feminist = yes
 		defensive_attrition = yes
-		max_consorts = 3
 		peace_prestige_loss = yes # Lose Prestige while at peace
 		raised_vassal_opinion_loss = no
 		allow_looting = yes
 		reformed = celtic_pagan_reformed
 		aggression = 1.5 # AI aggression factor
+		bs_marriage = yes
+		pc_marriage = yes
```

Comment:
Oh, Maelgwn also then went on to marry his nephew's wife. While not blood incest, this was *probably* (and this IS conjectural) political incest, given the Celts' views on the family as a single unit, his niece had become "part of the family" despite not being part of it by blood.
If you see what I mean?